backend/import/test_mo_api.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import mo_api
+import do_import
+
+
+def my_mo_get(url):
+    url_file = '.test_data/mo/' + url.replace('/', '#') + '.json'
+    with open(url_file, 'r') as f:
+        return json.load(f)
+
+
+def test_mo_data():
+    """Test the two classes derived from MOData."""
+    employee_uuid = '0014209a-8832-44b3-b7a0-4cc209a71993'
+    ou_uuid = '0418617c-242f-4d9a-81cc-abb269ad27b4'
+
+    ou = mo_api.MOOrgUnit(ou_uuid)
+    ou.get = my_mo_get
+    e = mo_api.MOEmployee(employee_uuid)
+    e.get = my_mo_get
+
+    # Test json function
+    assert(ou.json['name'] == 'Østervrå børnehus')
```

Comment:
`assert` is a keyword statement, like `print` used to be, not a function. The parentheses are redundant in this case.

Reply:
OK, will fix. Redundancy is evil. :-)